src/server/gc_worker/gc_worker.rs

```diff
@@ -2684,6 +2730,28 @@ mod tests {
             cfg,
         );
 
+        let cfg_tracker = {
+            let mut cfg = GcConfig::default();
+            // let ratio_threshold less than 1
+            cfg.ratio_threshold = 0.5;
+            cfg.enable_compaction_filter = true;
+            GcWorkerConfigManager(Arc::new(VersionTrack::new(cfg)))
+        };
+
+        let worker_builder = WorkerBuilder::new("gc-worker").pending_capacity(GC_MAX_PENDING_TASKS);
+        let worker = worker_builder.create().lazy_build("gc-worker");
```

Comment:
I'm having a hard time understand why this test works at all. The `worker` here will be destroyed at the end of this function.

Reply:
> I'm having a hard time understand why this test works at all. The `worker` here will be destroyed at the end of this function.

reworked.